docs/src/main/asciidoc/sqs.adoc

```diff
@@ -618,6 +618,11 @@ NOTE: Queues declared in the same annotation will share the container, though ea
 ===== SNS Messages
 
 Since 3.1.1, when receiving SNS messages through the `@SqsListener`, the message includes all attributes of the `SnsNotification`. To only receive need the `Message` part of the payload, you can utilize the `@SnsNotificationMessage` annotation.
+
+The @SnsNotificationMessage annotation can be used for both single and batch message processing.
+
+For single message processing, you can use the @SnsNotificationMessage annotation as follows:
```

Comment:
We avoid using `you` and and use a more formal tone instead.

So maybe something like `the @SnsNotificationMessage annotation can be used as follows:`?

Reply:
done